contribs/application/src/main/java/org/matsim/application/prepare/population/CleanPopulation.java

```diff
@@ -65,7 +65,7 @@ public Integer call() throws Exception {
 
 		// Using the analysis main mode identifier instead of the routing mode based one on purpose
 		// to be able to process older population files without any routing modes!
-		TripsToLegsAlgorithm trips2Legs = new TripsToLegsAlgorithm(new DefaultAnalysisMainModeIdentifier());
+		TripsToLegsAlgorithm trips2Legs = new TripsToLegsAlgorithm(new RoutingModeMainModeIdentifier());
```

Comment:
Hi, there is a comment above the line where DefaultAnalysisMainModeIdentifier() was replaced by RoutingModeMainModeIdentifier(). That comment still holds true for old (pre 2020) population files. Is this class used for old population files? Or is this for another use case?

Reply:
@rakow